packages/db/lib/gen-migration.mjs

```diff
@@ -0,0 +1,67 @@
+import fs from 'fs'
+import { stat } from 'fs/promises'
```

Comment:
since you already required 'fs/promises', try to use it everywhere in the code instead of fs.promises.*.

Reply:
Great point